build/images/ovs/README.md

```diff
@@ -17,5 +17,12 @@ cd build/images/ovs
 ./build.sh --pull --push
 ```
 
+To build Windows OVS image, you can follow these steps.
```

Comment:
@wenyingd @antoninbas Is it necessary to continue maintaining the image build method on the Windows host?
Previously, when I tried to test buildx on Windows, I found that it couldn't run directly based on Docker Engine. According to its doc(https://github.com/docker/buildx?tab=readme-ov-file#windows-and-macos), it requires Docker Desktop, which is unavailable for Windows Server OS. So I prefer to stop maintaining the previous image build method on Windows after this PR is merged. Do you have any other concerns or suggestions?

Reply:
A concern is the compatibility with the previous versions, e.g, new release for v2.0.1 or v1.5.2. If we switch to build on Linux with github actions, does it support to build the previous versions on Windows machine?

Reply:
> A concern is the compatibility with the previous versions, e.g, new release for v2.0.1 or v1.5.2. If we switch to build on Linux with github actions, does it support to build the previous versions on Windows machine? If no, then the original docker file was not designed for buildx on Linux, then it may fail.

This reminds me that for the CI testbed, we need to maintain the current topology until all active branches are switched to using the cross-compiled image.